<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RERestDish</class>
 <widget class="QDialog" name="RERestDish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dish</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabPage">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1" colspan="3">
          <widget class="EQLineEdit" name="leCode">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="PrimaryKey" stdset="0">
            <bool>true</bool>
           </property>
           <property name="Field" stdset="0">
            <string>f_id</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="EQLineEdit" name="leNameAm">
           <property name="Field" stdset="0">
            <string>f_en</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Description, am</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="btnBgColor">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resort/res.qrc">
             <normaloff>:/images/selection.png</normaloff>:/images/selection.png</iconset>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="3">
          <widget class="EQLineEdit" name="leQueue">
           <property name="Field" stdset="0">
            <string>f_queue</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="EColorLineEdit" name="leBgColor">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="Field" stdset="0">
            <string>f_bgColor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="EQLineEdit" name="leDefStore">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Default store</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="2">
          <widget class="EQLineEdit" name="leUnitName">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="1" colspan="3">
          <widget class="EQLineEdit" name="leADGT">
           <property name="Field" stdset="0">
            <string>f_adgt</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="EQLineEdit" name="leDefStoreName">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Last input price</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="EQLineEdit" name="leTypeName">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Unit</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>ArmSoft code</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="EColorLineEdit" name="leTextColor">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="Field" stdset="0">
            <string>f_textColor</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Min. reminder</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>ADGT</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QCheckBox" name="chAutoPrintTax">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="EQLineEdit" name="leTypeCode">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="Field" stdset="0">
            <string>f_type</string>
           </property>
           <property name="ShowButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QPushButton" name="btnTextColor">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resort/res.qrc">
             <normaloff>:/images/selection.png</normaloff>:/images/selection.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Code</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1" colspan="3">
          <widget class="EQLineEdit" name="leLastPrice"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Automaticaly print tax</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1" colspan="3">
          <widget class="EQLineEdit" name="leArmSoft">
           <property name="Field" stdset="0">
            <string>f_as</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="EQTextEdit" name="teAm">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="Field" stdset="0">
            <string>f_text_en</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="EQLineEdit" name="leUnitCode">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="EQLineEdit" name="leMinReminder">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="Field" stdset="0">
            <string>f_minreminder</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Queue</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Text color</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Background color</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Fiscal debt</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="EQLineEdit" name="leFiscalDebt">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="Field" stdset="0">
            <string>f_taxdebt</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Menu</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="tblMenu">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>MenuRecID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>MenuId</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Menu</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Print 1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Print 2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>StoreId</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Store</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Complex</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Required modifiers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnAddModifier">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Resort/res.qrc">
           <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblModifier">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <column>
          <property name="text">
           <string>RecId</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Image</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="btnLoadImage">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Resort/res.qrc">
           <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbImage">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../Resort/res.qrc">:/images/cancel.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Recipe</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnAppendRecipe">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resort/res.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnRemoveRecipeRow">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resort/res.qrc">
              <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnClearRecipe">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resort/res.qrc">
              <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>414</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chContainItself">
            <property name="text">
             <string>Contains itself</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="EQTableWidget" name="tblRecipe">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>Code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qty</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Scancode</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="EQLineEdit" name="leReadScancode"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstScancodes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnTrack">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/tracking.png</normaloff>:/images/tracking.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">eqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>EQTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">eqtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>EColorLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ecolorlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leDefStore</tabstop>
  <tabstop>leTypeCode</tabstop>
  <tabstop>leNameAm</tabstop>
  <tabstop>leUnitCode</tabstop>
  <tabstop>teAm</tabstop>
  <tabstop>leBgColor</tabstop>
  <tabstop>leTextColor</tabstop>
  <tabstop>leQueue</tabstop>
  <tabstop>leADGT</tabstop>
  <tabstop>leFiscalDebt</tabstop>
  <tabstop>leArmSoft</tabstop>
  <tabstop>leLastPrice</tabstop>
  <tabstop>chAutoPrintTax</tabstop>
  <tabstop>leMinReminder</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>leUnitName</tabstop>
  <tabstop>btnLoadImage</tabstop>
  <tabstop>btnAddModifier</tabstop>
  <tabstop>leDefStoreName</tabstop>
  <tabstop>tabPage</tabstop>
  <tabstop>tblModifier</tabstop>
  <tabstop>btnTrack</tabstop>
  <tabstop>btnAppendRecipe</tabstop>
  <tabstop>btnRemoveRecipeRow</tabstop>
  <tabstop>btnClearRecipe</tabstop>
  <tabstop>chContainItself</tabstop>
  <tabstop>tblRecipe</tabstop>
  <tabstop>leReadScancode</tabstop>
  <tabstop>lstScancodes</tabstop>
  <tabstop>tblMenu</tabstop>
 </tabstops>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
